app/assets/javascripts/admin/views/auth/registration_view.js

```diff
@@ -41,12 +43,12 @@ class RegistrationView extends React.PureComponent<Props, State> {
 
     this.props.form.validateFieldsAndScroll((err: ?Object, formValues: Object) => {
       if (!err) {
-        Request.sendJSONReceiveJSON("/api/register", { data: formValues }).then(()=>{
-           //if(...)
-           Toast.success(messages["auth.account_created"])
-           //else
-           //Toast.success(messages["auth.automatic_user_activation"])
-           app.router.navigate("/login", {trigger: true})
+        Request.sendJSONReceiveJSON("/api/register", { data: formValues }).then(() => {
+          // if(...)
```

Comment:
Whats going on here?

Reply:
I think @rschwanhold intend to show different Toasts for regular sign ups and when users are automatically activated. 

@normanrz Is there a still a config flag to auto. authenticate users without any email notifcation / admin unlock etc?

Reply:
I see. Yes, there still is auto signup and auto activation.